src/Controller/IntegratedPayment.php

```diff
@@ -144,4 +144,12 @@ public function already_updated(){
 		return true;
 	}
 
+	public static function display_notice() {
```

Comment:
We don't need to display WP or Wooco notices since we are using the popups on Vue lib

Reply:
done